### ISSUE TITLE
Questions about managing 000_CloudFormationResourceSpecification.json

### ISSUE THREAD

**Issue by jungseoklee** [awslabs/aws-cdk#658]
Hi AWS CDK,

I have questions about managing [000_CloudFormationResourceSpecification.json](https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/cfnspec/spec-source/000_CloudFormationResourceSpecification.json) file.

- How do you manage and update the file? Is it auto-generated from [AWS Resource Types Reference](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-template-resource-type-ref.html) document?
- How long does it take to reflect a new resource type or feature supported by CloudFormation?

Basically, I am interested in consistency between CloudFormation and CDK.

Thanks in advance!

**Comment by eladb** [comment]
The resource specification is published by CloudFormation in this location:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html
